installer/client/src/pages/WelcomeScreen.vue

```diff
@@ -0,0 +1,100 @@
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-form class="orangehrm-installer-page">
+    <oxd-text tag="h5" class="orangehrm-installer-page-title">
+      Welcome to OrangeHRM Stater Version 5.0 Setup Wizard
+    </oxd-text>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      This setup wizard guide through the steps necessary to install/upgrade
+      OrangeHRM Starter components and their dependencies.
+    </oxd-text>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      Select an installation type;
+    </oxd-text>
+
+    <oxd-form-row class="orangehrm-installer-page-row">
+      <oxd-radio-input
+        v-model="selected"
+        value="install"
+        option-label="Fresh Installation"
+      />
+      <oxd-text tag="p" class="orangehrm-installer-page-content --label">
+        Choose this option if you are installing OrangeHRM Starter for the first
+        time
+      </oxd-text>
+    </oxd-form-row>
+
+    <oxd-form-row class="orangehrm-installer-page-row">
+      <oxd-radio-input
+        v-model="selected"
+        value="upgrade"
+        option-label="Upgrading an Existing Installation"
+      />
+      <oxd-text tag="p" class="orangehrm-installer-page-content --label">
+        Choose this option if you are already using a prior version of OrangeHRM
+        Starter (version 2.6.5 to 4.8) and would like to upgrade to
+        <b>version 5.0</b>
+      </oxd-text>
+    </oxd-form-row>
+
+    <Notice title="important">
+      <oxd-text tag="p" class="orangehrm-installer-page-content">
+        Test
+      </oxd-text>
+    </Notice>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      Click <b>Next</b> to continue
+    </oxd-text>
+
+    <oxd-form-actions class="orangehrm-installer-page-action">
+      <oxd-button display-type="secondary" label="Next" type="submit" />
+    </oxd-form-actions>
+  </oxd-form>
+</template>
+
+<script>
+import Notice from "@/components/Notice.vue";
+import RadioInput from "@ohrm/oxd/core/components/Input/RadioInput";
```

Comment:
`import RadioInput from "@ohrm/oxd/core/components/Input/RadioInput";`
if the **oxd-components** are added locally or globally when it's used only for a single page?

Reply:
There are globally available components which are used commonly on many screen. its defined in `src/components.ts`
Some cases we only need specific components. here this radio-input component is not commonly used in other places therefor i imported it only for this page.